### PR TITLE
logging: correctly omits result.newoffset in Before

### PR DESCRIPTION
**Pull request by mathetake**

### CODE REVIEW

File: internal/wasip1/logging/logging.go [hunk 1]
```diff
@@ -157,6 +157,7 @@ func Config(fnd api.FunctionDefinition) (pSampler logging.ParamSampler, pLoggers
 			name = resultParamName(name)
 			logger = logMemI64(idx).Log
 			rLoggers = append(rLoggers, resultParamLogger(name, logger))
+			continue
```
Comment (mathetake) [inline internal/wasip1/logging/logging.go:160]: this was missing

File: imports/wasi_snapshot_preview1/fs_test.go [hunk 2]
```diff
@@ -2519,7 +2519,7 @@ func Test_fdSeek(t *testing.T) {
 				'?',
 			},
 			expectedLog: `
-==> wasi_snapshot_preview1.fd_seek(fd=4,offset=4,whence=0,4557430888798830399)
```
Comment (mathetake) [inline imports/wasi_snapshot_preview1/fs_test.go:2522]: just for anyone curious: basically this weird number is the u64 value on mem[newoffset]  _before_ we execute fd_seek.

Comment (mathetake) [reply imports/wasi_snapshot_preview1/fs_test.go:2522]: which is useless and meaningless in any way

**Comment by codefromthecrypt** [comment]
thanks!

**Pull request merged**